apps/code-server/4.95.3/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   code:
-    image: linuxserver/code-server:4.95.3
+    image: linuxserver/code-server:4.96.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The only difference between the two images is that one is `linuxserver/code-server:4.95.3` and the other is `linuxserver/code-server:4.96.1`. If you want to update to version 4.96.1, this change is necessary. There are no apparent irregularities or potential issues with the rest of the configuration.